platform/template/docker-syncd-base.mk

```diff
@@ -15,12 +15,18 @@ $(DOCKER_SYNCD_BASE)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
 $(DOCKER_SYNCD_BASE)_DBG_DEPENDS += $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_DEPENDS)
 
 $(DOCKER_SYNCD_BASE)_DBG_IMAGE_PACKAGES = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_IMAGE_PACKAGES)
-else
+else $(BLDENV),buster)
```

Comment:
ifeq

Reply:
Removed these changes, for the reason specified in the description